gpkitmodels/GP/aircraft/wing/wing_test.py

```diff
@@ -77,45 +75,10 @@ def box_spar():
         W, fs, perf, loading])
     m.solve(verbosity=0)
 
-def materials():
```

Comment:
why'd you remove the test?

Reply:
This tested that I could swap out the materials models to have a common material model for any use of CFRPFabric, but this became irrelevant once you import created models from the materials `__init__.py`

Reply:
ah, got it